test/util/network/network.go

```diff
@@ -100,7 +100,7 @@ func GetSriovNicIPs(pod *k8sv1.Pod, ifcName string) ([]string, error) {
 // Return a definition of a macvlan NetworkAttachmentDefinition
 // name name
 // namespace namespace
-func CreateMacvlanNetworkAttachmentDefinition(name string, namespace string, nodeNicName string) netattdefv1.NetworkAttachmentDefinition {
+func CreateMacvlanNetworkAttachmentDefinition(name string, namespace string) netattdefv1.NetworkAttachmentDefinition {
```

Comment:
please add in the comment here that because we are not give him any interface the host will select one, pointing to the doc please